Code/Tools/FBuild/FBuildWorker/FBuildWorkerOptions.cpp

```diff
@@ -19,18 +19,50 @@
 
 // FBuildWorkerOptions (CONSTRUCTOR)
 //------------------------------------------------------------------------------
-FBuildWorkerOptions::FBuildWorkerOptions() :
+FBuildWorkerOptions::FBuildWorkerOptions() 
+	: m_OverrideCPUAllocationWhenIdle( false )
+    , m_CPUAllocationWhenIdle( 0 )
+	, m_OverrideCPUAllocationDedicated(false)
+	, m_CPUAllocationDedicated(0)
 #if defined( __WINDOWS__ )
-    m_IsSubprocess( false ),
-    m_UseSubprocess( true ),
+	, m_IsSubprocess(false)
+	, m_UseSubprocess(true)
 #endif
```

Comment:
These properties were ordered to match the header, which is needed due to stricter warnings with Clang.